### DISCUSSION TITLE
Keyball44 トラックボールのトラブルシュート

### DISCUSSION THREAD

**Discussion by boxwood8** [Yowkees/keyball#388] [Q&A] [answered]
おかげさまでキーは全て動作するようになりましたが、ボールが反応しません。
下記の導通およびボール用ジャンパの導通は成功しています。
![image](https://github.com/Yowkees/keyball/assets/147312996/e7524b66-1f61-4348-83d9-5efd48a7ab5d)
また左右の ProMicro を入れ替えても変化が無いためボール基板の問題と想定しています。
ボール側基板の導通チェックについてピンの組み合わせをご教授ください。
![image](https://github.com/Yowkees/keyball/assets/147312996/25cf9d57-a216-4f0c-ace7-323fe664b39a)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

トラックボール基板の部品の向きや半田付けはお写真を見る限り問題なさそうです。

すでに導通チェックを行っていただいているようですので、L字7Pコンスルーのピンの変形をご確認ください。
下記写真のように少々の変形でも導通不良となる場合があります。

また、トラックボール基板がきちんと奥まで差さっているかもご確認ください（トラックボール基板の抜き差しのみで直る場合もあります）

以上、お手数ですがご確認お願いいたします。

![2_conthru_henkei](https://github.com/Yowkees/keyball/assets/84209772/e4e1ba6d-af5f-47a6-9c8e-597c8cada732)

**Reply by boxwood8** [reply]
ご対応ありがとうございました。

接触不良を疑いましたが、テスターで計測したところセンサー側まで電流が来ているため問題無さそうです。
![image](https://github.com/Yowkees/keyball/assets/147312996/412ab501-cfe2-4882-9a03-96a537c1a2a9)
ショップから改めてパーツ購入のご連絡をさせていただきますね

**Comment by Yowkees** [comment]
了解しました。ご連絡お待ちしております。
